runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx

```diff
@@ -2,13 +2,13 @@ import { VFC } from 'react';
 
 import styles from './EmbeddedVideo.module.scss';
 
-interface Props{
-    url: string
+interface Props {
+    url: string;
 }
 
-const EmbeddedVideo: VFC<Props> = ({url}) => (
+const EmbeddedVideo: VFC<Props> = ({ url }) => (
     <div className={styles.video}>
-        <iframe loading='lazy' className={styles.embed} src={url} />
+        <iframe loading="lazy" className={styles.embed} src={url} />
```

Comment:
i think eslint allows only for singlequote but if it does not cause compilation errors it's fine